cypress/commons/actions/generic/Scenarios.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { PAGE_NAME, URL_ROOT } from '../../TestConstants';
+import { SELECTORS } from '../../IdConstants';
+
+// Get elements in scenario view
+function getScenarioSelector () {
+  return cy.get(SELECTORS.scenario.selectInput);
+}
+function getScenarioCreationButton () {
+  return cy.get(SELECTORS.scenario.createButton);
+}
+function getScenarioCreationDialog () {
+  return cy.get(SELECTORS.scenario.createDialog.dialog);
+}
+function getScenarioCreationDialogNameField () {
+  return cy.get(SELECTORS.scenario.createDialog.nameTextfield);
+}
+function getScenarioCreationDialogMasterCheckbox () {
+  return cy.get(SELECTORS.scenario.createDialog.masterCheckbox);
+}
+function getScenarioCreationDialogDatasetSelector () {
+  return cy.get(SELECTORS.scenario.createDialog.datasetSelect);
+}
+function getScenarioCreationDialogParentScenarioSelector () {
+  return getScenarioCreationDialog()/* .click() */.find(SELECTORS.scenario.selectInput);
+}
+function getScenarioCreationDialogRunTypeSelector () {
+  return cy.get(SELECTORS.scenario.createDialog.typeSelect);
+}
+function getScenarioCreationDialogSubmitButton () {
+  return cy.get(SELECTORS.scenario.createDialog.submitButton);
+}
+
+// From scenario view, select the scenario with the provided name and id
+function select (scenarioName, scenarioId) {
```

Comment:
Actually the short name was set on purpose, because it is part of a "Scenarios" module, and will then be called with `Scenarios.select(...)`.
`Scenarios.selectScenario(...)` would be redundant I think